manifests/init.pp

```
@@ -40,6 +40,8 @@
 # $enable_ostree::      Boolean to enable ostree plugin. This requires existence of an ostree install.
 #                       type:boolean
 #
+# $pulp_admin_password:: password for the Pulp admin user. Leave blank for  random password to be generated
```

I think this should be 'Defaults to a random password' instead of leave blank.